packages/upload-client/src/util.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @typedef {import("@web3-storage/capabilities/src/types").SpaceDID} SpaceDID
+ */
+
+/**
+ * 
+ * @param {string} did 
+ * @returns {SpaceDID}
+ */
+export function ensureSpaceDID(did){
+  if (did.startsWith('did:key')) {
```

Comment:
This is more accurate IMO

```suggestion
  if (did.startsWith('did:key:')) {
```

Reply:
I think this is somewhat redundant, because it basically does what `SpaceDID.from` does see

https://github.com/web3-storage/ucanto/blob/c8999a59852b61549d163532a83bac62290b629d/packages/core/src/schema/type.ts#L26-L27

If we really want it perhaps we can define it as `export const  ensureSpaceDID = SpaceDID.from.bind(SpaceDID)` instead.

Reply:
oh awesome, replaced ensureSpaceDID with SpaceDID.from, thanks!